src/main/java/core/basesyntax/Buldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Buldozer extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Buldozer is work");

```

Comment:
The message 'Buldozer is work' is not informative enough and contains a grammatical mistake. It should be 'Bulldozer started its work' to clearly indicate that the Bulldozer is currently working.